### PR TITLE
Adds more detailed README for installation

### LINKED ISSUE #73: Create guidelines in Readme for getting started in the project
Currently the readme doesn't cover how to get started with Prisma, a new database and adding your own GitHub app. This needs to be added so that we can make it easier for contributors getting started. 

Bonus: This would be great as it's own page on the site too, and we will create a separate issue after we complete the guide. 

So we need to document:
- The required `.env` variables
  - `DATABASE_URL`
  - `GITHUB_ID`
  - `GITHUB_SECRET`
  - `NEXTAUTH_URL` (this could be included by default)
- Creating a new database (and getting a connection string) - Either a docker image or Supabase is handy. 
- How to setup the tables in database ` npx prisma db push`.
- Setting up a new GitHub app (getting the `GITHUB_ID` and a `GITHUB_SECRET`).

### PR DESCRIPTION
**Pull request by NiallJoeMaher**
# ✨ Codu Pull Request 💻

- Adds a much more detailed README.md
- Adds a default `.env.development` file
- Updates `.gitignore` to ignore new `.env.development` file

Closes #73

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +54,53 @@ Navigate to [http://localhost:3000](http://localhost:3000) in your browser to se
...
+
+The `DATABASE_URL` is a connection string to a PostgreSQL database (version 15.0). 
+
+By default, we point to a database running locally with Docker from our `docker-compose.yml` file.
+
+To run this file, make sure you have [Docker installed](https://docs.docker.com/get-docker/) and that Docker is running.
+  
+Run the command `docker compose up`.
+
+Alternatievly, if you have PostgreSQL running locally then you can use your local connection string or grab one from a free service like [Supabase](https://supabase.com/docs/guides/database/connecting-to-postgres#finding-your-connection-string).
```
Comment (CarolinaCobo) [inline README.md:73]: typo: Alternatievly > _Alternatively_

**Review by CarolinaCobo — APPROVED** [review]

**Review by gdomaradzki — APPROVED** [review]
LGTM

**Review by auxfuse — APPROVED** [review]
This looks really good 👏 Looking forward to getting my teeth into this soon 🙌

**Review by pkspyder007 — APPROVED** [review]
Looks good to go

**Pull request merged**